deployments/gpu-operator/templates/cleanup_crd.yaml

```diff
@@ -40,6 +40,11 @@ spec:
           - >
               kubectl delete clusterpolicy cluster-policy;
               kubectl delete crd clusterpolicies.nvidia.com;
-
+              kubectl delete crd nvidiadrivers.nvidia.com --ignore-not-found=true;
```

Comment:
We initially chose not to delete this CRD because it could be disruptive for users who may have deployed `NVIDIADriver` CRs out-of-band. Those CRs would get deleted when the CRD is deleted.  

However, I don't feel strongly about it anymore. @cdesiniotis @shivamerla to confirm